src/lib/index.module.css

```diff
@@ -1,166 +1,12 @@
 .layout {
-  flex: 1 1 auto;
-  overflow: hidden;
-  display: grid;
-  grid:
-      ".           .           ycount      .          " 1fr
-      "ylabel      yticks      data        xcount     " 2fr
-      ".           .           xticks      .          " min-content
-      ".           .           xlabel      .          " min-content
-    /  min-content min-content 2fr        1fr
-  ;
+  display: flex; 
+  flex: 1 1 auto; 
+  position: relative; 
+  overflow: hidden; 
+  padding: 10%;
```

Comment:
I don't recommend using percentage padding, I recommend just making it fixed or eliminated altogether. The user of this library can add their own padding as desired.